docs/content/concepts/assets/asset-checks/checking-for-data-freshness.mdx

```diff
@@ -0,0 +1,339 @@
+            yield ObserveResult(
+                asset_key=table_name,
+        metadata={"dagster/last_updated_timestamp": MetadataValue.timestamp(last_updated)},
+            )
+```
+
+### Step 2: Create a schedule for the observation
+
+Next, we'll define a [schedule](/concepts/partitions-schedules-sensors/schedules) that regularly executes the function in the `source_tables` observable source asset:
+
+```python
+from dagster import ScheduleDefinition
+
+source_tables_observation_schedule = ScheduleDefinition(
+  job=define_asset_job(
+    "source_tables_observation_job",
+    selection=AssetSelection.assets(source_tables),
+  ),
+  # Runs every minute. Usually, a much less frequent cadence is necessary,
+  # but a short cadence makes it easier to play around with this example.
+  cron_schedule="* * * * *",
+)
+```
+
+When the code location is loaded and the schedule is turned on, it will automatically kick off runs to observe the asset.
+
+### Step 3: Define the freshness check
+
+<TabGroup>
+<TabItem name="Option 1: Use hardcoded parameters">
+
+#### Option 1: Use hardcoded parameters
+
+In our example, we exepct the source tables to be updated no less than every two hours. We'll use the `build_freshness_checks` function to produce a set of asset checks that fail if an asset’s `last_updated_timestamp` is more than two hours before the current time:
+
+```python
+from dagster import build_freshness_checks_for_non_partitioned_assets
+
+source_table_freshness_checks = build_freshness_checks_for_non_partitioned_assets(
+  assets=[source_tables],
+  maximum_lag_minutes=2 * 60,
+)
+```
+
+These checks will automatically execute after the observations of the source assets they target, so an additional schedule isn't needed.
+
+</TabItem>
+<TabItem name="Option 2: Use time anomaly detection (Dagster Cloud Pro)">
+
+#### Option 2: Use time anomaly detection
+
+<Note>A Dagster Cloud Pro plan is required to use this feature.</Note>
+
+TODO: ADD NOTE ABOUT MATERIALIZATIONS AND ACCURACY
+
+Setting custom freshness policies on a large number of source assets can be time-consuming. Dagster Cloud Pro users can take advantage of a time series anomaly detection model instead of applying policies on an asset-by-asset basis. Freshness checks that use this approach function the same way checks with hardcoded parameters do.
+
+This model uses data from past materializations/observations to determine if data is arriving later than expected. **Note**: If the asset hasn't been updated enough times, the check will pass with a message indicating that more data is needed to detect anomalies.
+
+In the following example, we used `build_anomaly_detection_freshness_checks` to accomplish this:
+
+```python
+from dagster_cloud import build_anomaly_detection_freshness_checks
+
+freshness_checks = build_anomaly_detection_freshness_checks(
+  assets=[source_tables], params=None
+)
+```
+
+</TabItem>
+</TabGroup>
+
+### Step 4: Update the Definitions object
```

Comment:
```suggestion
### Step 4: Create a Definitions object
```

Reply:
^^suggested because there's not a preexisting Definitions object that they reupdating.